src/webhooks/pubsub/slackScores.test.ts

```diff
@@ -0,0 +1,114 @@
+import { bolt } from '@api/slack';
+import * as scoresUtils from '@utils/scores';
+
+import { triggerSlackScores } from './slackScores';
+
+describe('slackScores Tests', function () {
+  let getIssueEventsForTeamSpy, postMessageSpy;
+  beforeAll(() => {
+    getIssueEventsForTeamSpy = jest.spyOn(scoresUtils, 'getIssueEventsForTeam');
+    postMessageSpy = jest.spyOn(bolt.client.chat, 'postMessage');
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should handle case when no issues are returned', async () => {
+    getIssueEventsForTeamSpy.mockReturnValue([]);
+    await triggerSlackScores(null, null);
+    expect(postMessageSpy).toHaveBeenCalledWith({
+      blocks: [
+        {
+          text: {
+            emoji: true,
+            text: '🗓️ Weekly GitHub Response Times by Team 🗓️',
+            type: 'plain_text',
+          },
+          type: 'header',
+        },
+        {
+          text: {
+            text: `\`\`\`┌────────────────────────────────────────────────┐
```

Comment:
nit: I think we should have a leading newline in the output (maybe a trailing one too?), if for no other reason than that it makes tests easier to read ;)